test/integration/komodo_int_test.go

```diff
@@ -240,7 +240,6 @@ func (suite *SecretsIntegrationTestSuite) TestSecretsGet_EditorV0() {
 	ts.LoginAsPersistentUser()
 	cp := ts.Spawn("secrets", "set", "project.test-secret", "test-value", "--output", "editor.v0")
 	cp.ExpectExitCode(0)
-	suite.Empty(cp.TrimmedSnapshot())
```

Comment:
What was this doing?

Reply:
This was to verify that the output was empty, but it does not apply now that we use titles.

Reply:
But this is using editor.v0, so the title shouldn't be given?

Reply:
I see; Looks like a slip. The test is still passing with it added back in.